Hammerspoon Tests/HSserial.m

```diff
@@ -0,0 +1,45 @@
+//
+//  hs.serial Tests
+//
+
+#import "HSTestCase.h"
+
+@interface HSserial : HSTestCase
+
+@end
+
+@implementation HSserial
+
+- (void)setUp {
+    [super setUpWithRequire:@"test_serial"];
+}
+
+- (void)tearDown {
+    [super tearDown];
+}
+
+- (void)testAvailablePortNames {
+    RUN_LUA_TEST()
```

Comment:
I think these are all going to need a SKIP_IN_GITHUB_ACTIONS() before the RUN_LUA_TEST()s. It's awesome that you've added tests, but it looks like they depend on /dev/cu.MALS existing, which I'm guessing is the hardware you're using this with.

I've never been able to figure out what it is, but my Mac Pro seems to have a /dev/cu.pci-serial2, so maybe we can add the skips and then in the Lua test file, figure out which of the two serial devices to use - at least that way the test will pass on both your machine and mind :)

Reply:
`/dev/cu.MALS` should be built into macOS - do you not see it on your machine?

Reply:
I've changed to `Bluetooth-Incoming-Port`, which I'm HOPING is on most/all Mac's?

Reply:
yeah I have the bluetooth incoming one, but no MALS. Weird

Reply:
FWIW, I don't have the MALS one, either... what device does it correspond to?

Reply:
Maybe it's something to do with MacBook Pro's - connection to the Touch Bar?

Reply:
If so, then it's an intel thing as my M1 doesn't haven't it either (and it does have the touchbar). Oh well, I do think that the bluetooth one is probably more reliably available, so my curiosity about the device isn't that important 😜